functions/packages/feed-form/src/impl/types.ts

```diff
@@ -0,0 +1,35 @@
+export type YesNoFormInput = "yes" | "no" | "";
+export type AuthTypes =
+  | "0 - None"
+  | "API key - 1"
+  | "HTTP header - 2"
```

Comment:
[picky]: this pattern should be consistent across types, now we have`0 - None` and `API key - 1`. I'm suggesting to choose one of them and align the values, Example `None - 0` and `API key -1`.